test/pdfs/issue11045.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/pdf.js/files/3467425/from_pdftex.pdf
```

Comment:
Since this file is almost trivial, I'd prefer to have it in the repository instead of a linked testcase. I'll ask the author if we may include it.